File: sdkwork-backend-sdk-csharp/Models/SystemMessage.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class SystemMessage
    {
        public string? MessageType { get; set; }
        public Dictionary<string, object>? Metadata { get; set; }
        public string? Text { get; set; }
    }
}

File: sdkwork-backend-sdk-java/src/main/java/com/sdkwork/backend/model/SystemMessage.java
package com.sdkwork.backend.model;

public class SystemMessage {
    private String messageType;
    private Map<String, Object> metadata;
    private String text;

    public String getMessageType() {
        return this.messageType;
    }
    
    public void setMessageType(String messageType) {
        this.messageType = messageType;
    }

    public Map<String, Object> getMetadata() {
        return this.metadata;
    }
    
    public void setMetadata(Map<String, Object> metadata) {
        this.metadata = metadata;
    }

    public String getText() {
        return this.text;
    }
    
    public void setText(String text) {
        this.text = text;
    }
}
